docs/make.jl

```diff
@@ -20,6 +20,7 @@ makedocs(;
         canonical="stable",
     ),
     linkcheck=true,
+    analytics = "G-W1G68W77YV",
```

Comment:
The formatter prefers no whitespace here:

```suggestion
    analytics="G-W1G68W77YV",
```